src/main/logic/Logic.cpp

```diff
@@ -361,3 +375,19 @@ void Logic::gameEnd(){
 	mobileObjects.clear();
 	allObjects.clear();
 }
+
+void Logic::checkEnd(){
+	if(numLives <= 0){
+		bool end = TRUE;
```

Comment:
opss. Sorry that's due to in my work code we use TRUE and FALSE as they have been specifically declared somewhere.